argo-cd-apps/overlays/development/repo-overlay.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
     - resources-finalizer.argocd.argoproj.io
 spec:
   source: # This will be replaced with a reference to your fork of this repo (see hack/patch-apps-for-dev.sh)
-    repoURL: https://github.com/redhat-appstudio/infra-deployments.git
-    targetRevision: main
+    repoURL: https://github.com/chrisahl/infra-deployments.git
```

Comment:
Labeled as work-in-progress since it's not ready for merge when it's changing pointers to personal project. You can use preview mode, in that case you do not need to push the repoURL changes into MR.